specs/CustomScrollbar.md

```diff
@@ -0,0 +1,88 @@
+Custom Scrollbar 
+===
+
+# Background
+Developers would like to be able to set the scrollbar to Windows overlay style and match the Windows theme, but browser flags keep changing and are not reliable. Thus we want to provide an extendable API that's allow developers to define a scrollbar style in their app.
+
+# Examples
+## WinRT and .NET   
+```c#
+/// Create WebView Environment with option
+async void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.CustomScrollbarStyle = COREWEBVIEW2_SCROLLBAR_WIN_FLUENT_OVERLAY_STYLE;
```

Comment:
That's not what the enum should look like in .NET or WinRT langs. It should be CoreWebView2ScrollbarWinFluentOverlayStyle. Please try that change and make sure it compiles. This maybe works if you're using the .NET raw namespace or something?

Reply:
Updated  to use C# style